docs/site/contest_formats.md

```diff
@@ -28,16 +37,28 @@ The score is equal to the sum of scores of the last submissions to each problem.
 By default, ties are not broken, however, setting `cumtime` to `true` will sum the times of the last submission to each problem, and use that to tiebreak.
 
 The `first_ac_bonus` field, as suggested by the name, will add the specified number of points to a problem's score if it is solved on the first attempt, excluding compile errors and internal errors.
-This field defaults to `10`.
+This field defaults to 10.
 
 The `time_bonus` field awards a bonus for solving problems faster.
-The field is in minutes, and for each such interval of time before the contest ends, any submission with a non-zero score will have a bonus added. This field defaults to `5`.  
+The field is in minutes, and for each such interval of time before the contest ends, any submission with a non-zero score will have a bonus added. This field defaults to 5.  
 Note that specifying `0` will disable this bonus.  
 For example, say a submission with a score of `50/100` is submitted 23 minutes before the contest ends. $\left\lfloor\frac{23}{5}\right\rfloor = 4$, so 4 bonus points are awarded, giving a total score of $50 + 4 = 54$ for that problem.

```

Comment:
You can probably remove the code tag around 50/100 if you're going to clean this up :)